src/lib/components/load_failure_view.tsx

```diff
@@ -1,19 +1,17 @@
-import React from 'react'
+import * as React from 'react'
 import {
   requireNativeComponent,
 } from 'react-native'
 
-export default class LoadFailureView extends React.Component {
+export default class LoadFailureView extends React.Component<any,any> {
+  static propTypes = {
+    // A callback that is called when the user requests a retry.
+    onRetry: React.PropTypes.func,
+  }
```

Comment:
Ah right, forgot about that 👍 

This should probably be documented in the README:

* JS components: only a TS interface (which inherits from x and y)
* Native components: also add prop types